core/src/main/scala/better/files/Cmds.scala

```diff
@@ -124,4 +124,13 @@ object Cmds {
     } output.add(file, name.toString)
     destination
   }
+
+  def zip(files: (File, String)*)(destination: File, compressionLevel: Int)(implicit codec: Codec, d: DummyImplicit): File = {
+    for {
+      output <- new ZipOutputStream(destination.newOutputStream, codec).withCompressionLevel(compressionLevel).autoClosed
+      input <- files
+      file <- input._1.walk()
+    } output.add(file, input._2)
```

Comment:
Par curiosité, ça fait quoi si on a `name <- input._2.walk()` à l'intérieur du for suivi d'un `output.add(file, input._2)` ?